third_party/2and3/requests/models.pyi

```diff
@@ -114,6 +115,9 @@ class Response:
     def __bool__(self) -> bool: ...
     def __nonzero__(self) -> bool: ...
     def __iter__(self) -> Iterator[bytes]: ...
+    def __enter__(self) -> Response: ...
+    def __exit__(self, exc_type: Optional[type], exc_val: Optional[Exception],
```

Comment:
It actually just takes `*args`: https://github.com/requests/requests/blob/master/requests/models.py#L635. If you specify the arguments, the type should be `Optional[Type[BaseException]]` and the value should be `Optional[BaseException]`.

Reply:
This method is defined in PEP 343 and is supposed to accept those parameters. Now, if it is taking some short cuts... :-(